gen/definitions/bgp_peer.yaml

```diff
@@ -63,6 +64,24 @@ attributes:
     description: 'Source Interface. Must match first field in the output of `show intf brief`.'
     default_value: 'unspecified'
     example: 'lo0'
+  - nxos_name: holdIntvl
+    tf_name: hold_time
+    type: Int64
+    description: 'BGP Hold Timer in seconds. The value ust be greater than the keep alive timer'
+    default_value: 180
+    example: 5
+    omit_empty_value: true
+    min_int: 3
+    max_int: 3600
+  - nxos_name: kaIntvl
+    tf_name: keep_alive
```

Comment:
Could we maybe change this to `keepalive`?

Reply:
Sure can!